include/circt/Dialect/Calyx/CalyxLoweringUtils.h

```diff
@@ -293,7 +293,7 @@ class ComponentLoweringStateInterface {
   const DenseMap<unsigned, calyx::RegisterOp> &getBlockArgRegs(Block *block);
 
   /// Register 'grp' as a group which performs block argument
-  /// register transfer when transitioning from basic block from to to.
+  /// register transfer when transitioning from basic block 'from' to 'to'.
```

Comment:
Thanks for cleaning this stuff up. 

Do you have any thoughts on a canonical way that we should be doing documentation here? E.g., back-tick, single quote, double quote? I find I am very inconsistent.

Reply:
Not sure, to be honest, I'm inconsistent here as well.  Single quotes here chosen to match similar comment nearby.

Maybe backticks are the way to go?  Looking at this comment in the current doxygen:
https://circt.llvm.org/doxygen/classcirct_1_1calyx_1_1ComponentLoweringStateInterface.html#ad97349a3ac9c56af7d72578c1b7d0e96
The single quotes are rendered as themselves, but comparing to a random comment I found using backticks:
https://circt.llvm.org/doxygen/namespacecirct_1_1calyx_1_1direction.html#aa581977b67c4a52e186e2d320010f50f
They're styled as "code"/monospace, like in markdown.

Of course we could use things like `\p` for maximum doxygen goodness but that's less friendly to non-doxygen viewing, IMHO.

So, not sure but I think I'll try to use backtick in the future if there's no other competing reason (e.g., consistency).